components/custom-message.js

```diff
@@ -21,24 +21,22 @@ function labelText (image) {
 	return image ? 'Add an optional message to this chart' : 'Add an optional message';
 }
 
-export default ({customMessage, messageText, image, onMessageTyping, messageLength}) => {
-	if (customMessage) {
-		return (
-			<div className="email-article__message o-forms--wide">
-			<label className="email-article__message--label o-forms__label">{labelText(image)}
-			{characters(MAXIMUM_CHARACTERS, messageLength)}
-			</label>
-			{img(image)}
-			<textarea
-			className="o-forms__textarea email-article__message-textarea"
-			type="text"
-			inputMode="latin-prose"
-			maxLength={MAXIMUM_CHARACTERS}
-			placeholder="Enter your message"
-			rows="7"
-			value={messageText}
-			onChange={event => onMessageTyping(event.target.value)}></textarea>
-			</div>
-		)
-	}
-}
+export default ({messageText, image, onMessageTyping, messageLength}) => {
+	return (
```

Comment:
This can probably use the shorten version `() => (<div>...)` and remove the blank line